docs/content.md

```diff
@@ -2,54 +2,65 @@
 
 ## Upcoming 
 
+April 5th, 2022
+
+**AI4CI - Artificial Intelligence for Continuous Integration**
+
+* By Aakanksha Duggal, Hema Veeradhi, Karan Chauhan, Michael Clifford, Oindrilla Chatterjee and Shrey Anand
+* In this session, the speakers introduce AI4CI (Artificial Intelligence for Continuous Integration), a collection of open-source AIOps tools involving open operations data made available by Kubernetes testing and visualization platforms. 
+* The speakers will demonstrate how they developed a set of Jupyter Notebooks which are automated using Kubeflow Pipelines into repeatable processes that collect data from various open CI/CD data platforms, calculate and visualize key performance indicator metrics, build AI/ML services to support operations and create a reproducible workflow around these processes.
+* In this session, attendees learn about a set of AIOps services deployed on Operate First that they can start using and get familiarized with how to use the set of tools available on the Operate First cloud to deploy their own workloads.
+
+## Past Events
+
 March 22nd, 2022
 
 **Sandiego**
```

Comment:
nit: formatting for title and date for this session is different from others. should be on same line?